src/api/types.ts

```diff
@@ -239,6 +239,38 @@ interface GetCompaniesHouseInformationVariables {
   companiesHouseNumber: string;
 }
 
+interface Address {
+  addressLine1: string;
+  addressLine2?: string;
+  town?: string;
+  county?: string;
+  postalCode: string;
+}
```

Comment:
We could update `ApplicationCompanyAddress` here to extend from `Address`

Reply:
The standard properties could be used/extended now though, or if there are new differences from these changes can we address them now rather than creating duplicates/tech debt?

What are the differences? Maybe worth making some changes in `main-application` branch before go live.

Reply:
done